### PR TITLE
Sound component with s3 audio files

### PR DESCRIPTION
**Pull request by BrettBukowski**
Builds off of https://github.com/basillabs/Mission111/pull/44 and adds the s3 urls

### CODE REVIEW

File: app/components/Story.js [hunk 1]
```diff
@@ -154,18 +161,23 @@ class Story extends Component {
...
-    return <StoryCard
-              topContent={data.topContent}
-              bottomContent={data.bottomContent}
-              isSplit={this.state.isSplit}
-              onToggleTap={this.onClickToggle}
-              isTitleCard={data.isTitleCard}
-              allowLanguageSelection={data.allowLanguageSelection}
-              {...this.props}
-            />;
+  renderPage(data, index) {
```
Comment (Ako0) [inline app/components/Story.js]: where's this index used?

Comment (BrettBukowski) [inline app/components/Story.js]: removed!

**Comment by BrettBukowski** [comment]
@krapspark @Ako0 please take a look :)

**Comment by BrettBukowski** [comment]
Oh, nevermind, @Ako0 already approved! \o/

**Pull request merged**